config/consensus.go

```diff
@@ -1558,6 +1558,11 @@ func initConsensusProtocols() {
 	vAlpha5.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
 	Consensus[protocol.ConsensusVAlpha5] = vAlpha5
 	vAlpha4.ApprovedUpgrades[protocol.ConsensusVAlpha5] = 10000
+
+	// vFNetX are like vAlphaX but for AF's FNet
+	// vFnet1
+	vFnet1 := vFuture
```

Comment:
I'm aware re: future, the idea here was to update the code to fix `fnet1` to the current values for now, and when we need `fnet2` to shadow the next `future`, `fnet1` would be defined explicitly from v39 as you say.
The FNet nodes will track from this dev branch, so updates to future on main will not affect them until we update the branch

Reply:
I think you want to pin down fnet1 to last consensus version + the items you want to include, because you won't automatically include new vFuture items when the software gets upgraded. We're not upgrading it wily nily here, but if you rebase on master you'll have to tear down and rebuild the network. (This may be fine for your purposes.)

Reply:
Oh, this should _not_ target master, good catch @cce

Reply:
yep - sorry about that, changed the base to `dev/experimental-incentives`